pennylane_lightning/lightning_qubit/_measurements.py

```diff
@@ -133,6 +138,62 @@ def expval(self, measurementprocess: MeasurementProcess):
             measurementprocess.obs.name, measurementprocess.obs.wires
         )
 
+    # pylint: disable=protected-access
```

Comment:
Do we need that?

Reply:
I think because we're accessing a "private" attribute such as `self._measurement_lightning`.

Reply:
I don't think we need that when it is the class own private attribute.

Reply:
You're right, not required in `probs`.